spec-tests/src/test/java/org/candlepin/spec/pools/PoolSpecTest.java

```diff
@@ -565,4 +571,61 @@ public void shouldIncludeProductBrandingWhenFetchingPool() {
         assertTrue(b2filter.isPresent());
     }
 
+    @Test
+    @OnlyInStandalone
+    @DisplayName("should includes null values in json in Standalone")
+    public void shouldIncludesNullValuesInJsonInStandalone() throws ApiException {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO owner = this.createOwner(adminClient);
+        ApiClient ownerClient = this.createUserClient(adminClient, owner);
+
+        ProductDTO product = Products.randomSKU();
+
+        adminClient.ownerProducts().createProductByOwner(owner.getKey(), product);
```

Comment:
These two lines can be combined into one by using the in-class helper function for creating a product:

```
ProductDTO product = this.createProduct(adminClient, owner);
```